Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM php:7.1-cli
+
+ENV COMPOSER_ALLOW_SUPERUSER 1
+
+WORKDIR /code
+
+RUN apt-get update && apt-get install -y \
+        git \
+        unzip \
+   --no-install-recommends && rm -r /var/lib/apt/lists/*
+
+COPY ./docker/php/php.ini /usr/local/etc/php/php.ini
+
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin/ --filename=composer
+
+COPY composer.* ./
+RUN composer install $COMPOSER_FLAGS --no-scripts --no-autoloader
+COPY . .
```

Comment:
Nekopíruje se tam toho zbytečně moc?

Reply:
No, tak jako určitě :) Je otázka, jestli to nečemu vadí, standardně je všude jinde `COPY . /code`. Ve výsledku by se zrovna tady nemuselo kopírovat nic :) Je to jen na dev, takže to vždycky bude nalinkované. A testy se v tomhle případě pouští na infrastruktuře travisu, aby se dal udělat matrix verzí. Když tam nebude všechno, tak to bude křehčí a někdo bude muset myslet na to, aby tam přidal cokoli co přibude v projektu. Proto jsem v tomhle případě spíš proti.